test/v1/service.go

```diff
@@ -57,6 +57,19 @@ func validateCreatedServiceStatus(clients *test.Clients, names *test.ResourceNam
 	})
 }
 
+// GetServices get list of services available
+func GetServices(clients *test.Clients) (*v1.ServiceList, error) {
+	list, err := clients.ServingClient.Services.List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return list, nil
+}
+
+func DeleteService(clients *test.Clients, serviceName string) error {
+	return clients.ServingClient.Services.Delete(context.Background(), serviceName, metav1.DeleteOptions{})
+}
+
```

Comment:
@markusthoemmes just because they were shadowed for the creation and update. No other reason behind just following the same approach. I am not sure if I need to include more logic in those shadows either. I am happy to remove that if it is just adding noise.

Reply:
Ah, I see, I guess that makes sense. As such, you might want to wrap the actions into `reconciler.RetryTestErrors` then like the others are already.

Reply:
done